tools/swift-inspect/Sources/swift-inspect/RemoteMirror+Extensions.swift

```diff
@@ -64,11 +64,13 @@ extension SwiftReflectionContextRef {
     }
   }
 
-  internal func name(type: swift_reflection_ptr_t) -> String? {
+  internal func name(type: swift_reflection_ptr_t, mangled: Bool = false) -> String? {
     let typeref = swift_reflection_typeRefForMetadata(self, UInt(type))
     if typeref == 0 { return nil }
 
-    guard let name = swift_reflection_copyDemangledNameForTypeRef(self, typeref) else {
+    guard let name = if mangled { swift_reflection_copyMangledNameForTypeRef(self, typeref) } 
+                           else { swift_reflection_copyDemangledNameForTypeRef(self, typeref) }
```

Comment:
This is a bit awkward. What do you think about having a single `swift_reflection_copyNameForTypeRef` with a `mangled` parameter instead? We need to keep the old entrypoint around for other clients, but the new one could handle both cases.